.travis.yml

```diff
@@ -10,10 +10,17 @@ python:
   - '3.6'
 
 env:
-  - TEST_DATABASE_URL=postgresql://postgres@localhost:5432/pytest_test
+  - DB=pgsql TEST_DATABASE_URL=postgresql://postgres@localhost:5432/pytest_test
+  - DB=mysql TEST_DATABASE_URL=mysql+mysqldb://root@127.0.0.1/pytest_test
 
 addons:
   postgresql: '9.6'
+  mysql: '5.7'
+
+before_script:
+  - sh -c "if [ '$DB' = 'postgres' ]; then psql -c 'DROP DATABASE IF EXISTS pytest_test;' -U postgres; fi"
+  - sh -c "if [ '$DB' = 'postgres' ]; then psql -c 'CREATE DATABASE pytest_test;' -U postgres; fi"
+  - sh -c "if [ '$DB' = 'mysql' ]; then mysql -e 'CREATE DATABASE IF NOT EXISTS pytest_test;'; fi"
```

Comment:
Why are these lines necessary? I thought that the `database` fixture in `tests/_conftest.py` was setting up the database for us.

Reply:
Good point, I was learning a bit about travis when I copied these statements from the tutorial docs. Indeed they are being created in `_conftest.py`. I will remove them.